x/ccv/provider/keeper/throttle.go

```diff
@@ -431,8 +450,8 @@ func (k Keeper) GetSlashAndTrailingData(ctx sdktypes.Context, consumerChainID st
 
 // GetAllThrottledPacketData returns all throttled packet data for a specific consumer chain.
 //
-// Note: This method is only used by tests, hence why it returns redundant data as different types.
-// This method is not unit tested, as it is a test util.
+// Note: This method is only used by tests and queries, hence why it returns redundant data as different types.
+// Since this method executes on query, no panics are explicitly included.
```

Comment:
Should we return the error during the query? It seems like a cleaner approach than to just log error messages.

Reply:
What actually happens if we panic in a query?

Reply:
@MSalopek any views on this?

Reply:
Panics in queries could be dangerous, since it will bring down the node that tried to respond to the query.

However, simply logging an error will not communicate to the caller that there were issues during query execution.

What do you think about concatenating a bunch of errors and returning them? Invalid entries will never be returned so that may be a bit confusing (queue count is different than the number of returned entries).

Just an idea:
```golang
func (k Keeper) GetAllThrottledPacketData(ctx sdktypes.Context, consumerChainID string) (
	slashData []ccvtypes.SlashPacketData, vscMaturedData []ccvtypes.VSCMaturedPacketData, errors []error,
	rawOrderedData []interface{}, ibcSeqNums []uint64) {
        // ...
       if err != nil {
           errors = append(errors.New("err handling packet with sequence %d: err: %s, seq, err)
       }
```